src/_assets/javascripts/main.js

```diff
@@ -38,14 +38,16 @@ $(document).on('ready', function(){
   // Initiate Syntax Highlighting
   prettyPrint();
 
+  var highlightTimerId;
   // Frontpage footnotes
   function highlightFootnote() {
     var footnote = $('#code-display');
     footnote.removeClass('blink');
     footnote.addClass('blink');
-    window.setTimeout(function() {
+    if (highlightTimerId) clearTimeout(highlightTimerId);
+    highlightTimerId = setTimeout(function() {
       footnote.removeClass('blink');
-    }, 1000);
+    }, 2000);
```

Comment:
Why the increased time?

Reply:
It just made more sense with the new look. Since the commit that this change is a part of has been stashed, it will go away (and we can discuss it later, if need be).